### PR TITLE
update tests badge

### PR DESCRIPTION
**Pull request by mattnibs**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,11 +122,12 @@ at the [performance](performance/README.md) page.
...
-[ci-img]: https://circleci.com/gh/brimsec/zq.svg?style=svg
-[ci]: https://circleci.com/gh/brimsec/zq
-
 ## Join the Community
 
 Join our [Public Slack](https://join.slack.com/t/brimsec/shared_invite/zt-cy34xoxg-hZiTKUT~1KdGjlaBIuUUdg) workspace for announcements, Q&A, and to trade tips!
+
+[doc-img]: https://godoc.org/github.com/brimsec/zq?status.svg
+[doc]: https://godoc.org/github.com/brimsec/zq
+[tests-img]: ![Tests](https://github.com/brimsec/zq/workflows/Tests/badge.svg)
```
Comment (mikesbrown) [inline README.md:131]: Something is wrong with tests-img. See https://github.com/brimsec/zq/tree/tests-badge . Maybe just want the bare URL here?

Comment (mattnibs) [reply README.md]: I just fixed this

File: README.md [hunk 2]
```diff
@@ -122,11 +122,12 @@ at the [performance](performance/README.md) page.
...
-[ci]: https://circleci.com/gh/brimsec/zq
-
 ## Join the Community
 
 Join our [Public Slack](https://join.slack.com/t/brimsec/shared_invite/zt-cy34xoxg-hZiTKUT~1KdGjlaBIuUUdg) workspace for announcements, Q&A, and to trade tips!
+
+[doc-img]: https://godoc.org/github.com/brimsec/zq?status.svg
+[doc]: https://godoc.org/github.com/brimsec/zq
+[tests-img]: https://github.com/brimsec/zq/workflows/Tests/badge.svg
+[tests]: https://github.com/brimsec/zq/workflows/Tests
```
Comment (mikesbrown) [inline README.md:132]: Images look good now.

https://github.com/brimsec/zq/workflows/Tests is 404 for me. I was able to get something reasonable with https://github.com/brimsec/zq/actions?query=workflow%3ATests .

Comment (mattnibs) [reply README.md:132]: I should have checked this. Thanks!

**Comment by mikesbrown** [comment]
Thanks for noticing and fixing this, @mattnibs .

**Pull request merged**